postcss.config.js

```diff
@@ -1,7 +1,20 @@
+const autoprefixer = require('autoprefixer');
+const assets = require('postcss-assets');
+
+const CF_DIST = process.env.CLOUDFRONT_DIST;
```

Comment:
That is correct, but to @singingwolfboy's point, I'm not sure of an alternative approach that would allow for this to work. FWIW it is unlikely that we would create a new distribution in between deployments (or at all) on a given environment. The alternative would be to rely on posting these assets to S3 and relying on the S3->Cloudfront setup that I'm working in in #1922